src/libmongoc/src/mongoc/mongoc-linux-distro-scanner.c

```diff
@@ -67,7 +67,7 @@ _fgets_wrapper (char *buffer, size_t buffer_size, FILE *f)
        * This protects us from files like:
        * aaaaa...DISTRIB_ID=nasal demons
        */
-      TRACE ("Found line of length %ld, bailing out", len);
+      TRACE ("Found line of length %ld, bailing out", (long)len);
```

Comment:
Rather than casting to `long`, since we are building as C99 now, we can use `%zu` to format a `size_t`
```suggestion
      TRACE ("Found line of length %zu, bailing out", len);
```